DPU cannot be compiled

Using the source code at the latest commit (cb92f81f628d919b6ca7f5ebc30f9188a67c2ffb), when you try to compile the DPU, an error is raised:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project uv-e-skMartinContracts: Compilation failure: Compilation failure:
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skMartinContracts/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skmartincontracts/SkMartinContracts.java:[14,32] package org.apache.commons.lang3 does not exist
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skMartinContracts/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skmartincontracts/SkMartinContracts.java:[333,26] cannot find symbol
[ERROR] symbol:   variable StringUtils
[ERROR] location: class eu.comsode.unifiedviews.plugins.extractor.skmartincontracts.SkMartinContracts
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skMartinContracts/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skmartincontracts/SkMartinContracts.java:[338,87] cannot find symbol
[ERROR] symbol:   variable StringUtils
[ERROR] location: class eu.comsode.unifiedviews.plugins.extractor.skmartincontracts.SkMartinContracts
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skMartinContracts/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skmartincontracts/SkMartinContracts.java:[342,43] cannot find symbol
[ERROR] symbol:   variable StringUtils
[ERROR] location: class eu.comsode.unifiedviews.plugins.extractor.skmartincontracts.SkMartinContracts
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skMartinContracts/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skmartincontracts/SkMartinContracts.java:[420,25] cannot find symbol
[ERROR] symbol:   variable StringUtils
[ERROR] location: class eu.comsode.unifiedviews.plugins.extractor.skmartincontracts.SkMartinContracts
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skMartinContracts/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skmartincontracts/SkMartinContracts.java:[421,18] cannot find symbol
[ERROR] symbol:   variable StringUtils
[ERROR] location: class eu.comsode.unifiedviews.plugins.extractor.skmartincontracts.SkMartinContracts
```

The error looks like comsode-uv-plugins/e-skGovCRP#1.

--- a/pom.xml
+++ b/pom.xml
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>eu.unifiedviews</groupId>
 		<artifactId>uv-pom-dpu</artifactId>
-		<version>2.0.1</version>
+		<version>2.1.3</version>
 		<relativePath />
 	</parent>
 

--- a/pom.xml
+++ b/pom.xml
@@ -61,6 +61,10 @@
 			<artifactId>module-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+		</dependency>
 	</dependencies>
 
 	<repositories>

